core/tee/tee_ree_fs.c

```diff
@@ -138,6 +138,8 @@ struct block_operations {
 
 static struct handle_db fs_handle_db = HANDLE_DB_INITIALIZER;
 
+static struct mutex mutex = MUTEX_INITIALIZER;
```

Comment:
Even though the three different static variables `mutex` have nothing to do with each other and are static, it's still a bit confusing.

How about `ree_fs_mu` or `ree_fs_mutex` instead in this file for instance?